makefile.unix

```diff
@@ -228,56 +225,67 @@ src/hashes/sha2/sha512_224.o: src/hashes/sha2/sha512.c src/hashes/sha2/sha512_22
 src/hashes/sha2/sha512_256.o: src/hashes/sha2/sha512.c src/hashes/sha2/sha512_256.c
 src/hashes/sha2/sha256.o: src/hashes/sha2/sha256.c src/hashes/sha2/sha224.c
 
+#List of demo objects (only used to create a special rule to extend LTC_CFLAGS)
+DSOURCES != ls demos/*.c
+DOBJECTS = ${DSOURCES:.c=.o}
```

Comment:
I need to check this on HP-UX but I am afraid that it is too sophisticated for HP make